closed/OpenJ9.gmk

```diff
@@ -171,6 +171,11 @@ openj9_add_jdk_special = \
 	$(foreach target, $(JDK_OUTPUTDIR)/$(OPENJ9_BIN_OR_LIB_DIR) $(call FindLibDirForModule, java.base), \
 		$(eval $(call openj9_add_jdk_rules, $(target)/$(strip $1), $(OUTPUTDIR)/vm/$(strip $2))))
 
+# jitserver
```

Comment:
nit: Please add a blank line after this comment for consistency with your other pull requests.

Reply:
Done